library/set/add_spec.rb

```diff
@@ -1,5 +1,4 @@
 require_relative '../../spec_helper'
-require 'set'
```

Comment:
Probably we should `git mv library/set core/set` since it is effectively core now (and with https://bugs.ruby-lang.org/issues/21216 it's "fully" core but this effectively/fully distinction does not matter much for specs).
Could you do that in its own commit?

Reply:
Done.